src/aks-preview/HISTORY.md

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.5.42
+++++++
+* In AKS Monitoring addon, fix DCR resource naming convention from DCR-<workspaceName> to MSCI-<workspaceName> to make consistent naming across.
+
 0.5.42
```

Comment:
We already have 0.5.42 and a 0.5.43 ([PR #4054](https://github.com/Azure/azure-cli-extensions/pull/4054)) pending merge. Consider 0.5.44?

Besides, please also change the version [here](https://github.com/ganga1980/azure-cli-extensions/blob/main/src/aks-preview/setup.py#L11).

Reply:
> We already have 0.5.42 and a 0.5.43 ([PR #4054](https://github.com/Azure/azure-cli-extensions/pull/4054)) pending merge. Consider 0.5.44?
> 
> Besides, please also change the version [here](https://github.com/ganga1980/azure-cli-extensions/blob/main/src/aks-preview/setup.py#L11).

> @FumingZhang , fixed the feedback.